src/_components/RoutesComponent.jsx

```diff
@@ -24,12 +27,29 @@ import AddAssetSpec from './AssetSpecs/AddAssetSpecContainer';
 class RoutesComponent extends Component {
   checkAuthentication = () => !!(localStorage.getItem('art-prod-web-token'));
 
+  checkAdmin = () => {
+    const token = localStorage.getItem('art-prod-web-token');
+
+    if (token) {
+      const decodedToken = jwt.decode(token);
+
+      if (!has(decodedToken, 'admin')) {
```

Comment:
This `if` statement is unnecessary. [Lodash's has](https://lodash.com/docs/4.17.11#has) returns a boolean value. Therefore, it will always return `true` or `false`. The **token if block** can be compressed to:
```js
if (token) {
  return has(decodedToken, 'admin');
}
```
If the `admin` value will be available for all users, regardless if they are admins or not, then you can just return the `admin` value. Confirm this with Eric. If it is so, then the `checkAdmin` can be refactored to:
```js
checkAdmin = () => {
  const token = localStorage.getItem('art-prod-web-token');
  if (!token) {
    return false;
  }
 
 const decodedToken = jwt.decode(token);
 return decodedToken.admin;
}

Reply:
Okay. I will confirm with him